Facet/FacetField.php

```diff
@@ -45,7 +45,7 @@ public function __toString()
     /**
      * @return AttributeInterface
      **/
-    private function getFilter()
+    public function getFilter()
```

Comment:
Why has this been made public? Given that the implementation delegates most of its methods down to the filter, it shouldn't be necessary to expose a method that isn't on the interface that this class adheres to (and shouldn't be).

Reply:
OK I have an understanding of why, but don't agree with it. We shouldn't mutate visibility for this kind of reason, unless it's a design decision with appropriate changes to interfaces etc.